src/com/nutrons/nu17/subsystems/GearPlacer.java

```diff
@@ -0,0 +1,43 @@
+package com.nutrons.nu17.subsystems;
+
+import com.nutrons.nu17.RobotMap;
+
+//import edu.wpi.first.wpilibj.PIDSource;
+//import edu.wpi.first.wpilibj.PIDSourceType;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * @author Man Nguyen
+ */
+public class GearPlacer extends Subsystem {
+
+	public Talon gearPlacer = new Talon(RobotMap.PLACER);
+
+	public void initDefaultCommand() {
+		// Set the default command for a subsystem here.
+		// setDefaultCommand(new MySpecialCommand());
+	}
+
+	public void set(double position) {
+		gearPlacer.setPosition(position);
+	}
+
+	public double getPosition() {
+		return gearPlacer.getPosition();
+	}
+
+	/*
+	 * encoder unknown private class EncoderWrapper implements PIDSource {
```

Comment:
i got rid of it but i commented it out saying "encoder unknown" because we dont know if we'll have an encoder on the test bot yet